tests/sdk/sdk_integration_test.py

```
@@ -0,0 +1,106 @@
+import sys
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)